internal/web_server/web/views/pages/quiz_pages/user_profile.templ

```diff
@@ -0,0 +1,71 @@
+import "github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/icons"
+import "github.com/Molnes/Nyhetsjeger/internal/models/users"
+
+//Javascript function to show the delete confirmation modal
+
+script showDeleteConfirmationModal() {
+    document.addEventListener("htmx:confirm", function(e) {
+
+        if (e.target.id === "delete-button") {
+
+        e.preventDefault();
+        document.getElementById("modal").classList.remove("hidden");
+        document.getElementById("modal").classList.add("flex");
+
+        document.getElementById("accept-button").addEventListener("click", function() {
+            document.getElementById("modal").classList.add("hidden");
+            document.getElementById("modal").classList.remove("flex");
+            e.detail.issueRequest(true)
+        });
+
+        document.getElementById("refuse-button").addEventListener("click", function() {
+            document.getElementById("modal").classList.add("hidden");
+            document.getElementById("modal").classList.remove("flex");
+        }); 
+    }
+    })
+}
+
+templ UserProfile(u *users.User) {
+	@layout_components.BaseLayout("Nyhetsjeger - Profil") {
+		@showDeleteConfirmationModal()
+		<div class="h-screen flex justify-center items-center">
+			<form class="w-1/3 h-3/4 bg-purple-400 rounded-lg flex flex-col items-center" action="" method="post">
+				<h1 class="text-3xl my-4 font-bold">Oppdater profil</h1>
+				<div class="w-3/4">
+					<div>
+						<label for="tlf-in" class="inline font-bold mr-4">Mobilnummer</label>
+						<button class="inline rounded-full border-gray-700 text-gray-700 border-2 w-6 h-6 font-bold mb-1" type="button">i</button>
+					</div>
+					<input id="tlf-in" class="w-full h-7 rounded px-2" type="tel" placeholder="123 45 678" value={ u.Phone } pattern="^(\\d{2} \\d{2} \\d{2} \\d{2}|\\d{3} \\d{2} \\d{3}|\\d{8})$" name="phonenumber" required/>
+					<label for="usrn-in" class="font-bold">Brukernavn</label>
+					<p id="usrn-in" class="w-full h-7 rounded px-2 bg-white" type="text">{ u.Username }</p>
+				</div>
+				<button id="randomize" class="flex items-center m-2" type="button" hx-patch="/api/v1/quiz/brukernavn" hx-trigger="click" hx-target="#usrn-in">
+					Generer nytt navn
+					@icons.Dice(16, "currentColor", 28, 28)
+				</button>
+				<div class="flex items-center">
+					<input id="comp-check" class="inline w-6 h-6" type="checkbox" name="competition"/>
+					<p for="comp-check" class="inline ml-2">Bli med i konkurranser</p>
+				</div>
+				<input class="mb-4 mt-5 py-2 px-8 bg-white border-2 rounded-lg font-bold" type="submit" value="Lagre"/>
+				<button hx-post="/auth/logout" class="mb-4 mt-5 py-1 px-8 bg-white border-2 rounded-lg font-bold" type="button">Logg ut</button>
+				<a class="mt-5" href="#">Terms of services</a>
+				<button id="delete-button" hx-delete="" hx-confirm="Sikker på at du ønsker å slette brukeren?" class="text-red-600 font-bold mt-auto mb-4" type="button">SLETT KONTO</button>
```

Comment:
Why is there an empty hx-delete? Also, duplicate confirmation text, but it is not the same text. `"Sikker på at du ønsker å slette brukeren?"` and `Er du sikker på at du ønsker å slette din konto? Dette kan ikke angres!` If this is unintentional, consider making a variable that is used both places.